.github/workflows/release.yaml

```diff
@@ -22,7 +22,9 @@ env:
   PACKAGE_NAME: {{project-name}}
   # The itch.io page to upload to, in the format: `user-name/project-name`.
   # Comment this out to disable.
+  {%- if itch_username != "" -%}
```

Comment:
Can we instead comment it out? That way, a user can still activate itch support later

Reply:
I can do an else statement that has the same text, but commented out if you want. That would be the best solution